app/code/Magento/Bundle/Model/Plugin/Frontend/ProductIdentitiesExtender.php

```diff
@@ -37,12 +37,15 @@ public function __construct(Type $type)
      */
     public function afterGetIdentities(CatalogProduct $product, array $identities): array
     {
+        if ($product->getTypeId() !== BundleType::TYPE_CODE) {
+
```

Comment:
Not sure if this is right method to ask a question, but I will shoot:
I see that these lines are added to composite type of products `if ($product->getTypeId() !== BundleType::TYPE_CODE) {`, which means it will never find the parent product from child product. Is that intentional? If so then how will we invalidate parent cache upon child product update?